doc/user_manual/OutStreamSystem.tex

```diff
@@ -262,7 +262,11 @@ \subsubsection{\textbf{DataObjects} Printing}
   ``second-example'')
   \item \textbf{Input}, the input space will be dumped out (see
   ``third-example'')
-  \item \textbf{Input|var-name-in/Output|var-name-out}, only the particular
+  \item \textbf{metadata}, the metadata will be dumped out. This information
```

Comment:
can we use a synonym for "dumped"? such as "saved to disk", "printed to file" or similar?

Reply:
I have updated it.